lib/src/models/config_models/scroll_to_bottom_button_config.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/cupertino.dart';
+
+/// Configuration for the "Scroll to Bottom" button.
+class ScrollToBottomButtonConfig {
+  ScrollToBottomButtonConfig({
+    this.backgroundColor,
+    this.border,
+    this.borderRadius,
+    this.icon,
+    this.scrollAnimationDuration,
+    this.alignment,
+    this.padding,
+    this.onButtonClick,
+    this.buttonDisplayOffset,
+  });
+
+  /// The background color of the button.
+  final Color? backgroundColor;
+
+  /// The border of the button.
+  final Border? border;
+
+  /// The border radius of the button.
+  final BorderRadius? borderRadius;
+
+  /// The icon displayed on the button.
+  final Icon? icon;
+
+  /// The duration of the scroll animation when the button is clicked.
+  final Duration? scrollAnimationDuration;
+
+  /// The alignment of the button on top of text field.
+  final Alignment? alignment;
```

Comment:
We can create separate enum similar to `SuggestionListAlignment`. According to the current implementation, right, center, and only left alignments will work, with the button always at the bottom of the screen.

```
enum SuggestionListAlignment {
  left(Alignment.bottomLeft),
  center(Alignment.bottomCenter),
  right(Alignment.bottomRight);

  const SuggestionListAlignment(this.alignment);

  final Alignment alignment;
}
```